src/App.js

```diff
@@ -6,11 +6,43 @@ import './App.css';
 function App() {
 
      const[inputText ,setInputText] = useState('');
-     const[todos,setTodos] = useState([])
+     const[todos,setTodos] = useState([]);
+
+     // Done Multiple
+     const doneMultipleHandler = () => {
+         setTodos( todos.map(item => {
+          if(item.check){
+            return ({
+              ...item,completed :!item.completed
+            })
+          }
+          else {
+            return ({
+              ...item, completed : item.completed
+            })
+          }
+          return item 
```

Comment:
You are already returning in `if` and `else` then no need to return the item here.
Also, you can make it better by removing  else statement and doing something like
```
if(condtion){
   return ...
}
return ...
```